### PR TITLE
Set app binary info through config

### PR DESCRIPTION
**Pull request by jeongchanKim**

### CODE REVIEW

File: loadable_apps/loadable_sample/Kconfig [hunk 1]
```diff
@@ -106,3 +106,6 @@ endif #EXAMPLES_MICOM_TIMER_TEST
 
 endif #EXAMPLES_LOADABLE_AUTOMATIC_TEST
 endif #EXAMPLES_LOADABLE
+
+source "/root/tizenrt/loadable_apps/loadable_sample/micomapp/Kconfig"
+source "/root/tizenrt/loadable_apps/loadable_sample/wifiapp/Kconfig"
```
Comment (sunghan-chang) [inline loadable_apps/loadable_sample/Kconfig:111]: source loadable_apps/loadable_sample/micomapp/Kconfig
source loadable_apps/loadable_sample/wifiapp/Kconfig

File: loadable_apps/loadable_sample/micomapp/Kconfig [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+		App1 Binary version(YYMMDD format)
+
+config APP1_DYN_RAMSIZE
+	int "App1 binary Dynamic RAM size"
+	default 512000
+	---help---
+		App1 Binary Dynamic RAM size
+
+config APP1_STACKSIZE
+	int "App1 binary Stacksize"
```
Comment (sunghan-chang) [inline loadable_apps/loadable_sample/micomapp/Kconfig:38]: Binary Stack size?
Maybe main task stack size?

File: loadable_apps/loadable_sample/micomapp/Kconfig [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+	default 4096
+	---help---
+		App1 Binary Stacksize
+
+config APP1_PRIORITY
+	int "App1 binary Priority"
```
Comment (sunghan-chang) [inline loadable_apps/loadable_sample/micomapp/Kconfig:44]: same as previous

File: os/tools/mkbinheader.py [hunk 4]
```diff
@@ -201,15 +201,14 @@ def make_kernel_binary_header():
...
-    kernel_ver = sys.argv[4]
-    binary_name = sys.argv[5]
-    binary_ver = sys.argv[6]
-    dynamic_ram_size = sys.argv[7]
-    main_stack_size = sys.argv[8]
-    main_priority = sys.argv[9]
-    comp_enabled = sys.argv[10]
-    comp_blk_size = sys.argv[11]
-    loading_priority = sys.argv[12]
+    binary_name = sys.argv[4]
```
Comment (jeongarmy) [inline os/tools/mkbinheader.py:204]: I think we need to update descriptions in above comments together, line 190~198. Please update them.

File: loadable_apps/loadable_sample/Kconfig (hunk 1 above)
Comment (jeongchanKim) [reply loadable_apps/loadable_sample/Kconfig:111]: Changed. Thanks.

File: loadable_apps/loadable_sample/micomapp/Kconfig (hunk 2 above)
Comment (jeongchanKim) [reply loadable_apps/loadable_sample/micomapp/Kconfig:38]: Added "Main". Thanks

File: loadable_apps/loadable_sample/micomapp/Kconfig (hunk 3 above)
Comment (jeongchanKim) [reply loadable_apps/loadable_sample/micomapp/Kconfig:44]: Added "Main". Thanks

File: loadable_apps/loadable_sample/micomapp/Kconfig [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+		App1 Binary version(YYMMDD format)
+
+config APP1_DYN_RAMSIZE
+	int "App1 binary Dynamic RAM size"
+	default 512000
+	---help---
+		App1 Binary Dynamic RAM size
+
+config APP1_MAIN_STACKSIZE
+	int "App1 binary Main Stacksize"
```
Comment (sunghan-chang) [inline loadable_apps/loadable_sample/micomapp/Kconfig:38]: How about `Main Task Stacksize of App1 Binary`?
I think description should say full words.

File: os/tools/mkbinheader.py (hunk 4 above)
Comment (jeongchanKim) [reply os/tools/mkbinheader.py:203]: Done.

File: loadable_apps/loadable_sample/micomapp/Kconfig (hunk 5 above)
Comment (jeongchanKim) [reply loadable_apps/loadable_sample/micomapp/Kconfig]: Modified.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**